### ISSUE TITLE
SHA256 sum failed Linux binary (no high priority or security implications) 

### ISSUE THREAD

**Issue by MCM-Mike** [mimblewimble/grin-wallet#359]
When you automatically checking the SHA256 sum of a binary we encountered a problem. When checking with the usual "sha256sum" Linux tools you will receive an error due to a directory in checksum file:

error:
`sha256sum: /home/vsts/work/1/a/grin-wallet-v3.1.0-linux-amd64.tar.gz: No such file or directory
/home/vsts/work/1/a/grin-wallet-v3.1.0-linux-amd64.tar.gz: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
`

**sha256 checksum file**
`SHA256(/home/vsts/work/1/a/grin-wallet-v3.1.0-linux-amd64.tar.gz)= bdc0ea8b8163bee7265ed63ee525720f3ad511b984987189a719427362b00313
`
Could you upload only the filename and the associate hash value in order to check this automatically?

**Comment by jaspervdm** [comment]
I can't reproduce
```
$ sha256sum grin-wallet-v3.1.0-linux-amd64.tar.gz
bdc0ea8b8163bee7265ed63ee525720f3ad511b984987189a719427362b00313  grin-wallet-v3.1.0-linux-amd64.tar.gz
```

**Comment by MCM-Mike** [comment]
This is why I did mention the hash itself is correct but if you use the automated check by:

`_$ sha256sum -c grin-wallet-v3.1.0-linux-amd64-sha256sum.txt 

sha256sum: /home/vsts/work/1/a/grin-wallet-v3.1.0-linux-amd64.tar.gz: No such file or directory
/home/vsts/work/1/a/grin-wallet-v3.1.0-linux-amd64.tar.gz: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
`
Both files and sha256sum and tar.gz are in the same directory when doing this check. 

You will get an error because you did set a directory within the sha256sum file "/home/vsts/..." where the tar.gz file should be located.

**Comment by jaspervdm** [comment]
Oh i see what you mean, didn't know about the `- c` flag. Will see if i can fix it!

**Comment by MCM-Mike** [comment]
You can fix this by just change the grin-wallet-v3.1.0-linux-amd64-sha256sum.txt content to:

`SHA256(grin-wallet-v3.1.0-linux-amd64.tar.gz)= bdc0ea8b8163bee7265ed63ee525720f3ad511b984987189a719427362b00313
`

This allows us to check the file in the same directory.

**Comment by MCM-Mike** [comment]
push, as its sill not fixed with latest release 3.1.2
